main.py

```diff
@@ -31,22 +30,18 @@ async def start(_, update: Message):
 
 
 @Bot.on_message(filters.command(["count"]))
-async def count(_, update: Union[Message, CallbackQuery], count=0):
-    text = f"Total {str(count)} clicks"
-    reply_markup = InlineKeyboardMarkup(
-        [[InlineKeyboardButton(text="Click Here", callback_data="count=" + str(count))]]
+async def count(_, update: Message):
+    await update.reply_text(
+        text="Total 0 clicks",
+        reply_markup=InlineKeyboardMarkup(
+            [[InlineKeyboardButton(text="Click Here", callback_data="count")]]
+        ),
     )
-    if isinstance(update, CallbackQuery):
-        await update.answer(text="Added your click.\n\n" + text, show_alert=True)
-        await update.message.edit_text(text=text, reply_markup=reply_markup)
-    else:
-        await update.reply_text(text=text, reply_markup=reply_markup)
 
 
 @Bot.on_message(filters.command("reset"))
 async def reset_count(_, update: Message):
-    if (reply := update.reply_to_message) and (mkp := reply.reply_markup):
-        mkp.inline_keyboard[0][0].callback_data = "count=0"  # modifying the callback data
```

Comment:
Does that actually work without this line?

Reply:
yes callback data is just "count" now, current count is automatically parsed from message.text.